Src/Fido2.Models/CredentialCreateOptions.cs

```diff
@@ -136,6 +136,7 @@ public IReadOnlyList<PublicKeyCredentialHint> Hints
                 PubKeyCredParam.ES512,
                 PubKeyCredParam.RS512,
                 PubKeyCredParam.PS512,
+                PubKeyCredParam.RS1

```

Comment:
P-8 Send a valid ServerAuthenticatorAttestationResponse with SELF "packed" attestation, for "ALG_SIGN_RSASSA_PKCSV15_SHA1_RAW" aka "RS1" algorithm, and check that server succeeds‣

Reply:
Ideally, this should be made configurable instead of being hard coded, with defaults excluding RS1 (IMHO).

Reply:
Tracked in #533, resolving this conversation.